packages/client/hmi-client/src/components/mathml/tera-math-editor.vue

```diff
@@ -225,9 +225,6 @@ const hideKeyboard = () => {
 watch(
 	() => props.latexEquation,
 	() => {
-		if (props.latexEquation === '') {
-			isEditingEquation.value = true;
-		}
```

Comment:
Why removing this?

Reply:
added this back in and found a way around this, I didnt want the editor to pop up in the equations here when they were an empty string